be/src/vec/columns/column_dictionary.h

```diff
@@ -294,19 +294,14 @@ class ColumnDictionary final : public COWHelper<IColumn, ColumnDictionary<T>> {
             return -1;
         }
 
-        inline int32_t find_code_by_bound(const StringValue& value, bool lower, bool eq) const {
+        inline int32_t find_code_by_bound(const StringValue& value, bool greater, bool eq) const {
             auto code = find_code(value);
             if (code >= 0) {
                 return code;
             }
-
-            if (lower) {
-                return std::lower_bound(_dict_data.begin(), _dict_data.end(), value) -
-                       _dict_data.begin() - eq;
-            } else {
-                return std::upper_bound(_dict_data.begin(), _dict_data.end(), value) -
-                       _dict_data.begin() + eq;
-            }
+            auto bound = std::upper_bound(_dict_data.begin(), _dict_data.end(), value) -
```

Comment:
Need some comment to make here easier to understand

Reply:
done